test/pjrt/test_experimental_pjrt_multi_cpu.py

```diff
@@ -108,6 +108,23 @@ def test_spawn(self, spawn):
     indices = sorted(queue.get(block=False) for _ in range(queue.qsize()))
     self.assertListEqual(indices, list(range(4)))
 
+  @staticmethod
+  def _hlo_dump(tmpdir: str):
+    os.environ['XLA_SAVE_TENSORS_FMT'] = 'hlo'
+    os.environ['XLA_SAVE_TENSORS_FILE'] = os.path.join(tmpdir, 'save.hlo')
+
+    x = torch.randn((3, 3), device=xm.xla_device())
+    xm.mark_step()
+    x.cpu()
+
+  def test_hlo_dump(self):
```

Comment:
can we add the same test under maybe `test_operations`? I want to make sure this change does not break non-pjrt work loads. If it is too complicated we can do it in a separate pr.

Reply:
This is actually already tested on XRT under this test case: https://github.com/pytorch/xla/blob/master/test/dynamo/test_dynamo_graph_dump.py

I know that because it caught a bug in the name of my files :smile:

Reply:
lol OK, I forgot I added that test.